schema.graphql

```diff
@@ -41,6 +42,29 @@ type Block @entity {
   supplierParams: [SupplierParam] @derivedFrom(field: "block")
   tokenomicsParams: [TokenomicsParam] @derivedFrom(field: "block")
   consensusParams: [ConsensusParam] @derivedFrom(field: "block")
+  # REPORTS
+  totalComputedUnits: BigInt
+  totalRelays: BigInt
+  failedTxs: Int
+  validTxs: Int
+  totalTxs: Int
+  stakedSuppliers: Int
+  suppliersStakedTokens: BigInt
+  unstakingSuppliers: Int
+  suppliersUnstakingTokens: BigInt
+  took: Int
```

Comment:
What is this - is there a more semantically meaningful name that we can use instead?

Reply:
took is how much time (in ms) the block took to be produced. In fact this is the subtract of the timestamp of the current block against the previous one.

Reply:
suggestions for this:
1. `blockIntervalTime`
2. `timeToBlock`